auth/src/test/java/io/grpc/auth/GoogleAuthLibraryCallCredentialsTest.java

```diff
@@ -191,8 +192,9 @@ public void invalidBase64() throws Exception {
   }
 
   @Test
-  public void credentialsFailsWithIoException() throws Exception {
-    Exception exception = new IOException("Broken");
+  public void credentialsFailsWithRetryableRetryableException() throws Exception {
+    boolean retryable = true;
+    Exception exception = new RetryableException(retryable);
```

Comment:
Any reason to not to just say `new RetryableException(true)` ?

Reply:
Because `isRetryable() == true`.

Reply:
Yes, I kind of suspected that - but wondering since `new RetryableException(/*retryable=*/ true)` was the existing pattern.